### DISCUSSION TITLE
Enable/Disable backend

### DISCUSSION THREAD

**Discussion by AlwaysHC** [dotnet/yarp#202] [General] [answered]
I need a solution to enable and disabled backends when, for instance, I'm deploying new versions or when I need to get down a server

**Comment by AlwaysHC** [comment]
Currently I use this commands for nginx

sed -i "s/server $PROXY:3000 down;/server $PROXY:3000;/g" /etc/nginx/conf.d/default.conf
nginx -s reload

sed -i "s/server $PROXY:3000;/server $PROXY:3000 down;/g" /etc/nginx/conf.d/default.conf
nginx -s reload

**Comment by Tratcher** [comment, marked as answer]
See the related discussion over at https://github.com/microsoft/reverse-proxy/issues/190
- Health check notifications to temporarily suspend a destination server
- OR Rolling upgrades by removing a destination server from the config and then re-adding it later.

**Reply by samsp-msft** [reply]
Commenting the destination out in the config is probably the easiest - the config file is watched for changes, so you don't need to restart the proxy for changes to take effect.
